packages/lesswrong/components/comments/CommentPermalink.tsx

```diff
@@ -121,7 +125,12 @@ const CommentPermalink = ({
           />
         )}
         <div className={classes.seeInContext}>
-          <a href={`#${documentId}`}>See in context</a>
+          <a href={`#${documentId}`} onClick={(e) => {
+            if (!hasInContextComments) return;
+
+            scrollFocusOnElement({ id: comment._id, options: { behavior: "smooth" } });
+            e.preventDefault()
+          }}>See in context</a>
```

Comment:
With the new system, linking to `#${documentId}` wouldn't change the scrollToCommentId, so wouldn't trigger a re-scroll from our code, but would trigger the browser level scroll (which doesn't have as nice spacing). So I'm manually re-scrolling here.

The `e.preventDefault()` isn't necessary, I just think it's good to move in the direction of not having these two separate links if it defaults to in-context anyway (but there are other places in the code that still use the hash, and I haven't tried to change all of them)

Reply:
(Also I decided to keep the comment at the top, in case you still manage to lose it)